cdragontoolbox/export.py

```diff
@@ -422,6 +424,7 @@ def _create_exporter(self, patch):
             ImageConverter(('.dds', '.tga')),
             BinConverter(re.compile(r'game/.*\.bin$'), btype_version),
             SknConverter(),
+            RstConverter(re.compile(r'game/data/menu/.*\.(txt|stringtable)$'))
```

Comment:
`data/menu/bootstrap_en_us.stringtable` for example, in `Bootstrap.wad.client`.
I believe they added those after version 2 of RST, aka the binary format.
I guess `.stringtable` is the "correct" extension, but because they kept the original (.txt) ones and didn't rename them, we must use this somewhat ugly heuristic to catch the `data/menu/fontconfig_en_us.txt` and similar as well.